lib/api_valve/forwarder.rb

```diff
@@ -94,6 +94,7 @@ def faraday
         url: endpoint,
         ssl: {verify: false}
       ) do |config|
+        config.request :instrumentation
```

Comment:
Can you pass additional options to set the instrumentation name to `apivalve.forwarder`? By default it uses `faraday.request` which is a bit generic and might conflict with other requests.

See https://github.com/lostisland/faraday_middleware/blob/master/lib/faraday_middleware/instrumentation.rb#L27

Reply:
Actually thinking about it it might make most sense to leave it as the default for whoever wants to hook into this.